proxies/live/apiproxy/proxies/default.xml

```diff
@@ -92,7 +92,7 @@
   </RouteRule>
   <RouteRule name="sync-wrap">
     <Condition>
-      request.verb = "PATCH" AND (proxy.pathsuffix MatchesPath "/Patient/{id}") AND request.header.x-sync-wrapped NotEquals "true"
+       request.header.x-sync-wrapped NotEquals "true" AND (request.verb = "PATCH" AND (proxy.pathsuffix MatchesPath "/Patient/{id}")) OR (request.verb = "POST" AND (proxy.pathsuffix MatchesPath "/Patient"))
```

Comment:
```suggestion
       request.header.x-sync-wrapped NotEquals "true" AND ((request.verb = "PATCH" AND (proxy.pathsuffix MatchesPath "/Patient/{id}")) OR (request.verb = "POST" AND (proxy.pathsuffix MatchesPath "/Patient")))
```

Reply:
You might need to use some extra round brackets, A AND B OR C != A AND (B OR C)
Might be possible to test with a POST /Patient without the x-sync-wrapped header

Reply:
Good spot. Interesting. I would have thought there would be an infinite loop in the tests if that were true. But makes sense to make it explicit.